### DISCUSSION TITLE
00_pytorch_fundamentals  GPU False RTX 3090

### DISCUSSION THREAD

**Discussion by AI-IntegratedSolutions** [mrdbourke/pytorch-deep-learning#613] [General]
Only on chapter 0 and having issues :)  Getting a false on GPU, I'm a newb, using a RTX 3090 on ROG Zephyrus M16.  Can someone help?  Thanks:

```python
import pandas as pd
import numpy as np
import torch
import sklearn
import matplotlib
import torchinfo, torchmetrics

# Check PyTorch access (should print out a tensor)
print(torch.randn(3, 3))

# Check for GPU (should return True)
print(torch.cuda.is_available())
```

```
tensor([[-0.3895,  0.3402, -0.0611],
        [-0.1007,  2.1187, -0.3073],
        [ 2.1518,  0.2172,  1.5166]])
False
```

**Comment by mrdbourke** [comment]
Hi @AI-IntegrationSolutions , 

Did you go through with the GPU installation of PyTorch? See: https://pytorch.org/get-started/locally/ 

Be sure to select "CUDA": 

<img width="1624" alt="Screenshot 2023-09-07 at 10 38 18 am" src="https://github.com/mrdbourke/pytorch-deep-learning/assets/16750345/9d0e7c44-4b2a-4a16-987e-12e463152bd5">

Getting CUDA setup on local machines is a bit of a challenge when you first do it (I've done it a few times and have spent several hours troubleshooting it).

Here are my notes on setting up a new GPU on a new PC (they are rough but they contain some helpful resources), see: https://www.notion.so/mrdbourke/Installing-CUDA-on-a-new-GPU-175916438b814525b8b161ac0c787667?pvs=4

**Reply by AI-IntegratedSolutions** [reply]
Finally got it working, thanks.  Apparently i have a 3070 and not 3090.  And i had a 2022 model, not the 2023 advertised on Amazon, screwed again.
